stage1/aci/aci-install.mk

```diff
@@ -95,6 +96,7 @@ $(AMI_GEN_MANIFEST): $(AMI_SRC_MANIFEST) | $(AMI_TMPDIR)
 		-e 's/@RKT_STAGE1_NAME@/$(AMI_SED_NAME)/g' \
 		-e 's/@RKT_STAGE1_VERSION@/$(AMI_SED_VERSION)/g' \
 		-e 's/@RKT_STAGE1_ENTER@/$(AMI_SED_ENTER)/g' \
+		-e 's/@RKT_STAGE1_STOP@/$(AMI_SED_STOP)/g' \
```

Comment:
Please update lines [103](https://github.com/jjlakis/rkt/blob/e9338c6ce34252e7c535272da51e2335d1c84493/stage1/aci/aci-install.mk#L103) and [104](https://github.com/jjlakis/rkt/blob/e9338c6ce34252e7c535272da51e2335d1c84493/stage1/aci/aci-install.mk#L104) accordingly, too.